README.md

```diff
@@ -22,31 +25,23 @@ This repository contains the Puppet modules for WSO2 Open Banking.
     b. Reassign the *$jdk_name* variable in `<puppet_environment>/modules/ob_common/manifests/params.pp` to the name of the downloaded JDK distribution.
 
 3. Change the hostnames in `<puppet_environment>/modules/ob_common/manifests/params.pp` of **Puppetmaster**.
-
-4. Update the `<puppet_environment>/modules/ob_common/manifests/params.pp` file of **Puppetmaster** according to the Open 
-   Banking specification (UK, AU, Berlin). The default value is set as UK.
    
-5. Set up the databases in the **Puppet agent** by following the [Configuring Databases](https://docs.wso2.com/display/OB200/Configuring+Databases+for+UK) documentation. <br>
+4. Set up the databases in the **Puppet agent** by following the [Configuring Databases](https://ob.docs.wso2.com/en/latest/install-and-setup/setting-up-databases) documentation. <br>
    Update the `<puppet_environment>/modules/ob_common/manifests/params.pp` file of **Puppetmaster** to point the created databases.
 
-6. Run the following profiles on the **Puppet agent**, by executing the following commands.
-
-    a. To run the ```Open Banking Business Intelligence Dashboard profile```:
-    
-        export FACTER_profile=obbi_dashboard
-        puppet agent -vt
+5. Run the following profiles on the **Puppet agent**, by executing the following commands.
 
-    b. To run the ```Open Banking Business Intelligence Worker profile```:
+    a. To run the ```Open Banking Business Intelligence profile```:
     
-        export FACTER_profile=obbi_worker
+        export FACTER_profile=obbi
         puppet agent -vt
         
-    c. To run the ```Open Banking Identity & Access Management profile```:    
+    b. To run the ```Open Banking Identity & Access Management profile```:
```

Comment:
For OB3.0 we use the base product name, so I think we can use
"Open Banking Identity Serve profile"

Reply:
Please check in step 6 also.

Reply:
Done